LuaRules/Configs/target_lookahead_defs.lua

```diff
@@ -0,0 +1,11 @@
+-- unitDefID -> lookahead
+local unitArray = {}
+
+for i=1,#UnitDefs do
+	local cp = UnitDefs[i].customParams
+	if cp and cp.lookahead then
+		unitArray[i] = cp.lookahead
+	end
```

Comment:
Can drop the `cp` check, all defs are supposed to have a valid customparams table (even if empty)
```suggestion
	unitArray[i] = UnitDefs[i].customParams.lookahead
```